config_test_imdb.yml

```diff
@@ -0,0 +1,21 @@
+experiment_name: test_imdb
+n_repeats: 1
+scenario_params_list:
+ - dataset_name:
+    - 'imdb'
+   partners_count:
+    - 3
+   amounts_per_partner: 
+    - [0.2, 0.5, 0.3]
+   samples_split_option:
+    - ['advanced', [[1, 'shared'], [1, 'shared'], [1, 'specific']]]
```

Comment:
The use of advance split options are not very clear for me, so just to be sure : here you only have two classes, so the two first partners share some sample of class 0 and class 1, and  the last partner has only class 1, am I right?

Reply:
Yes, there is only two differents label to caracterize the reviews: positive or negative. But the advanced split options doesn't work like this. The data samples from the initial dataset are split in cluster per data labels.
From the README.txt file: `'advanced' approach [[nb of clusters (int), 'shared' or 'specific']]`
Partners 1 and 2 share a cluster that contains samples of class 0, Partner 3 has the cluster with the samples of class 1.

There is a line in scenario.py to prevent wrong config.yml:
In Scenario.py: ligne 297: `assert specific_clusters_count + shared_clusters_count <= nb_diff_labels`

Reply:
Ok its a bit clearer now, thanks